bootstrap-scripts/package_install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -v
+
+set -ex
+if [ '$pnda_mirror$' != '$''pnda_mirror''$' ]; then 
```

Comment:
Yes it is because the aws and heat repo are kept separate. 
This was an attempt to keep both file copies identical to ease code maintenance, but I agree at the expense of code simplicity.
I'll take it out and defer those changes to when/if both repo's merge together.